src/icons/fileIcons.ts

```diff
@@ -2050,7 +2050,16 @@ export const fileIcons: FileIcons = {
       fileNames: ['maven.config', 'jvm.config', 'pom.xml'],
     },
     { name: 'ada', fileExtensions: ['ada', 'adb', 'ads', 'ali'] },
-    { name: 'serverless', fileNames: ['serverless.yml'] },
+    {
+      name: 'serverless',
+      fileNames: [
+        'serverless.yml',
+        'serverless.yaml',
+        'serverless.json',
+        'serverless.js',
```

Comment:
I was just wondering if the files `serverless.js` and `serverless.ts` are official config icons? Are these files which are commonly used within the community? Not sure about this. I'd rather have an icon for only these three files:

'serverless.yml',
        'serverless.yaml',
        'serverless.json',

Reply:
I believe TypeScript configuration is quite widely used by community . Although the documentation doesn't cover it extensively, it was officially released in version [v1.72](https://github.com/serverless/serverless/releases/tag/v1.72.0). There's also a [TS library](https://github.com/serverless/typescript#serverlesstypescript) available that contains all the necessary types for the Serverless configuration under the Serverless organization.

Reply:
Thank you for your reply. If it was even officially introduced, then of course we leave it in. That's ok.